### ISSUE TITLE
Reflect use of new hadolint organisation

### ISSUE THREAD

**Issue by zemanlx** [hadolint/hadolint#121]
* [ ] This is a bug report
* [x] This is a feature request
* [x] I searched existing issues before opening this one

1. Codebase 
    - find and replace mostly as a result of [search](https://github.com/hadolint/hadolint/search?utf8=%E2%9C%93&q=%22lukasmartinelli%2Fhadolint%22&type=)
2. Docker Hub 
    - there should be also versioned image of hadolint (for each following release)
    - make README more docker-hub-markdown friendly
3. CI
    - make sure that Travis and AppVeyor are on and working

**Comment by zemanlx** [comment]
@lukasmartinelli I will need your help with 3. as you are the owner of the organisation. I have already requested access for Travis and you probably need to add me to AppVeyor to the hadolint organisation.

**Comment by lukasmartinelli** [comment]
> @lukasmartinelli I will need your help with 3. as you are the owner of the organisation. I have already requested access for Travis and you probably need to add me to AppVeyor to the hadolint organisation.

I changed everyone to owner so now u should have access.

> and you probably need to add me to AppVeyor to the hadolint organisation.

So https://ci.appveyor.com/project/lukasmartinelli/hadolint now uses hadolint/hadolint as source.
I don't know whether you meant creating orgs on travis and appveyor (is that possible?).

**Comment by zemanlx** [comment]
Thank you @lukasmartinelli!

- Travis is easy as it is syncing with GitHub so that one is up and running at https://travis-ci.org/hadolint/hadolint
- AppVeyor is a tricky one - I had to create an account and then rename it to hadolint. They have feature _teams_ which can sync GitHub teams with AppVeyor so I have created team _Appveyor Admins_ where I have added just you and me. and anybody else can add themselves if they are interested. Looks like that that one is also live https://ci.appveyor.com/project/hadolint/hadolint/branch/master
